iOS/Collection View Management /DiffableCollectionViewController.swift

```diff
@@ -85,6 +84,7 @@ class DiffableCollectionViewController<SectionType: Hashable,
         }
 
         self.collectionView.animationView.stop()
+        self.handleDataBeingLoaded()
```

Comment:
Now called whenever the data is reloaded. I think before we only wanted it called once, but now we have reconfigure so.

Reply:
Makes sense